Extensions/DialogueTree/bondage.js/dist/bondage.min.js

```diff
@@ -8,4 +8,4 @@ text:"",token:null,line:this.yylineno})},lex:function(){var e=this.next();return
pleDefaultsInSwitch),i=!0),t.push(n);return ft.inSwitch=r,F("}"),lt.createSwitchStatement(e,t)}function Te(){var e;return D("throw"),$()&&M({},tt.NewlineAfterThrow),e=he(),U(),lt.createThrowStatement(e)}function Pe(){var e,t;return lt.markStart(),D("catch"),F("("),B(")")&&R(ht),e=de(),ot&&h(e.name)&&j({},tt.StrictCatchVariable),F(")"),t=pe(),lt.markEnd(lt.createCatchClause(e,t))}function $e(){var e,t=[],n=null;return D("try"),e=pe(),G("catch")&&t.push(Pe()),G("finally")&&(T(),n=pe()),0!==t.length||n||M({},tt.NoCatchOrFinally),lt.createTryStatement(e,[],t,n)}function Me(){return D("debugger"),U(),lt.createDebuggerStatement()}function je(){var e,t,n,r=ht.type;if(r===He.EOF&&R(ht),lt.markStart(),r===He.Punctuator)switch(ht.value){case";":return lt.markEnd(xe());case"{":return lt.markEnd(pe());case"(":return lt.markEnd(be())}if(r===He.Keyword)switch(ht.value){case"break":return lt.markEnd(Ce());case"continue":return lt.markEnd(Ae());case"debugger":return lt.markEnd(Me());case"do":return lt.markEnd(Se());case"for":return lt.markEnd(ke());case"function":return lt.markEnd(De());case"if":return lt.markEnd(_e());case"return":return lt.markEnd(Oe());case"switch":return lt.markEnd(Le());case"throw":return lt.markEnd(Te());case"try":return lt.markEnd($e());case"var":return lt.markEnd(ge());case"while":return lt.markEnd(we());case"with":return lt.markEnd(Ie())}return e=he(),e.type===Qe.Identifier&&B(":")?(T(),n="$"+e.name,Object.prototype.hasOwnProperty.call(ft.labelSet,n)&&M({},tt.Redeclaration,"Label",e.name),ft.labelSet[n]=!0,t=je(),delete ft.labelSet[n],lt.markEnd(lt.createLabeledStatement(e,t))):(U(),lt.markEnd(lt.createExpressionStatement(e)))}function Re(){var e,t,n,r,i,o,s,a,c=[];for(lt.markStart(),F("{");st<ut&&ht.type===He.StringLiteral&&(t=ht,e=Ge(),c.push(e),e.expression.type===Qe.Literal);)n=it.slice(t.range[0]+1,t.range[1]-1),"use strict"===n?(ot=!0,r&&j(r,tt.StrictOctalLiteral)):!r&&t.octal&&(r=t);for(i=ft.labelSet,o=ft.inIteration,s=ft.inSwitch,a=ft.inFunctionBody,ft.labelSet={},ft.inIteration=!1,ft.inSwitch=!1,ft.inFunctionBody=!0;st<ut&&!B("}")&&(e=Ge(),"undefined"!=typeof e);)c.push(e);return F("}"),ft.labelSet=i,ft.inIteration=o,ft.inSwitch=s,ft.inFunctionBody=a,lt.markEnd(lt.createBlockStatement(c))}function Fe(e){var t,n,r,i,o,s,a=[];if(F("("),!B(")"))for(i={};st<ut&&(n=ht,t=de(),o="$"+n.value,ot?(h(n.value)&&(r=n,s=tt.StrictParamName),Object.prototype.hasOwnProperty.call(i,o)&&(r=n,s=tt.StrictParamDupe)):e||(h(n.value)?(e=n,s=tt.StrictParamName):l(n.value)?(e=n,s=tt.StrictReservedWord):Object.prototype.hasOwnProperty.call(i,o)&&(e=n,s=tt.StrictParamDupe)),a.push(t),i[o]=!0,!B(")"));)F(",");return F(")"),{params:a,stricted:r,firstRestricted:e,message:s}}function De(){var e,t,n,r,i,o,s,a,c=[];return lt.markStart(),D("function"),n=ht,e=de(),ot?h(n.value)&&j(n,tt.StrictFunctionName):h(n.value)?(o=n,s=tt.StrictFunctionName):l(n.value)&&(o=n,s=tt.StrictReservedWord),i=Fe(o),c=i.params,r=i.stricted,o=i.firstRestricted,i.message&&(s=i.message),
```

Comment:
.min.js files are usually coming from another repository/are auto generated, so they should not be modified. Can you revert the changes so they disappear from the PR? Thanks!